src/gt4sd/configuration.py

```diff
@@ -28,9 +28,8 @@
 from functools import lru_cache
 from typing import Dict, Optional, Set
 
-from pydantic import BaseSettings
-
 from .s3 import GT4SDS3Client, S3SyncError, sync_folder_with_s3, upload_file_to_s3
+from pydantic_settings import BaseSettings, SettingsConfigDict
```

Comment:
is isort fine with this?

Reply:
Is isort still used? Seems to be commented out in `dev_requirements.txt`?

Reply:
We removed it in #218 @drugilsberg. 
Still good practice to have absolute imports ahead of relative ones @fiskrt

Reply:
thanks for the inputs, agree with Jannis